### ISSUE TITLE
Aboslute path on win10 don't work well

### ISSUE THREAD

**Issue by ToMattBan** [crunchy-labs/crunchy-cli#289]
**Describe the bug**
Trying to download Dr Stone, passing the full path of my folder, it don't work.
It instead create a "D" folder where I'm executing the script.
So, instead of the anime going to "D:\Animes\Dr. Stone\S01E01.mkv" it goes to "C:\iam\executing\in\this\folder\D\Animes\Dr. Strone\S01E01.mkv"
I believe this is a problem just with Windows.

**To Reproduce**
```
$ crunchy archive -a ja-JP -m audio -o "D:\Animes\Dr. STONE\S01 E01 - New World.mkv" https://www.crunchyroll.com/series/GYEXQKJG6/[S1E1]
```

**Expected behavior**
To download the anime at the right path

**Client (please complete the following information):**
 - OS: Win10
 - Version 3.1.1

**Comment by Dokman** [comment]
i've tryed it and it's not working well with drives and folders

**Comment by bytedream** [comment]
Fixed in [v3.2.0](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.2.0)
